app/controllers/invites_controller.rb

```diff
@@ -20,8 +20,8 @@ def accept
       set_website(@invite.website)
       # it will redirect to errors url because we set the website for the user
       after_login_redirect
-    else
-      # it will redirect to create website wizard because we don't sent it
+    elsif @invite.email.casecmp(current_user.email) == 1
+      # it will redirect to website wizard url because emails don't match
       after_login_redirect
```

Comment:
In short there is no need for this check to see if the email is different as it already happens.